src/sage/symbolic/ginac/numeric.cpp

```diff
@@ -51,6 +51,7 @@
 
 #define PY_SSIZE_T_CLEAN
 #include <Python.h>
+#include "gmp.h"
```

Comment:
Feel like flint bug to me? Surely the header should be usable standalone?

Reply:
I think it's by design: With
https://github.com/flintlib/flint/commit/3604bb2e4b630bc5a8104d17451badf972810979 one needs to include the gmp header before using `fmpz_get_mpz`.